.github/workflows/nightly_tests.yaml

```diff
@@ -89,6 +89,12 @@ jobs:
       run : python3 xpk.py info --cluster $TPU_CLUSTER_NAME --zone=us-central2-b
     - name: Delete the workload on the cluster
       run: python3 xpk.py workload delete --workload $WORKLOAD_NAME --cluster $TPU_CLUSTER_NAME --zone=us-central2-b
+    - name: Run a batch job on the cluster
+      run: python3 xpk.py batch --cluster $TPU_CLUSTER_NAME --zone=us-central2-b test.sh
+    - name: Get job info for the last job created on the cluster
+      run: |
+        export MOST_RECENT_JOB_NAME=$(kubectl get jobs --sort-by=.metadata.creationTimestamp -o jsonpath="{.items[-1:].metadata.name}") 
```

Comment:
can items be empty?

Reply:
It shouldn't be, since the previous test (get job list) would fail if it is. I will also add the "get job list" test to nightly tests for this reason.